weekly-updates/weekly-update.2015-05-15.md

```diff
@@ -0,0 +1,26 @@
+# io.js News
+
+We are happy to speak this news, **"io.js decides to join the Node Foundation"**. And our organization name will be renamed `nodejs`.
+
+As a first step, we will move from `iojs` organization to `nodejs` organization and will converge `joyent/node` gradually. We will continue to release `io.js` until the convergence have done.
+
+If you would like to see the convergence plan in detail, please check it at [jashkell repo](https://github.com/jasnell/dev-policy/blob/master/convergence.md). When you would like to ask questions, feel free to ask [@Fishrock123](mailto:fishrock123@rocketmail.com). The discussion log is [here](https://github.com/nodejs/io.js/issues/1664). 
+
+Please wait for new node release a little more :)
+
+# io.js release
+This week we hadn't io.js release.
+
+### Community Updates
+
+* [Performance Showdown](https://raygun.io/blog/2015/05/performance-showdown-node-js-vs-io-js-v2-0-0/): Node.js vs. io.js v2.0.0.
+* Microsoft will propose Node run different JavaScript backend including `Chakra`.
```

Comment:
Microsoft has node.js running on their JavaScript VM `Chakra`.

And we should probably link to the repo.